### PR TITLE
Add Gif library

### PR DESCRIPTION
**Pull request by sgara**

### CODE REVIEW

File: cmake/find/Findgiflib.cmake [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# ----------------------------------------------------------------------
+# Findodb.cmake - odb CMake module
```
Comment (ruslo) [inline cmake/find/Findgiflib.cmake]: Please fix this one.

File: cmake/projects/giflib/hunter.cmake [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    SHA1
+    5f1157cfc377916280849e247b8e34fa0446513f
+)
+
+hunter_configuration_types(giflib CONFIGURATION_TYPES Release)
+hunter_pick_scheme(DEFAULT url_sha1_autotools)
+hunter_cacheable(giflib)
+hunter_download(
+    PACKAGE_NAME giflib
+    PACKAGE_INTERNAL_DEPS_ID "2"
```
Comment (ruslo) [inline cmake/projects/giflib/hunter.cmake]: `PACKAGE_INTERNAL_DEPS_ID "2"` can be removed.

File: cmake/find/Findgiflib.cmake (hunk 1 above)
Comment (ruslo) [inline cmake/find/Findgiflib.cmake]: Please fix comment.

File: cmake/projects/giflib/hunter.cmake [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2015, Damien Buhl
```
Comment (ruslo) [inline cmake/projects/giflib/hunter.cmake]: Please fix `Copyright`

File: docs/packages/pkg/giflib.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
+.. spelling::
+
+    foo
+
+.. index:: unsorted ; giflib
+
+.. _pkg.foo:
```
Comment (ruslo) [inline docs/packages/pkg/giflib.rst]: -> `pkg.giflib`

File: docs/packages/pkg/giflib.rst [hunk 5]
```diff
@@ -0,0 +1,19 @@
+.. spelling::
+
+    foo
+
+.. index:: unsorted ; giflib
```
Comment (ruslo) [inline docs/packages/pkg/giflib.rst]: `unsorted` -> `media`?

File: docs/packages/pkg/giflib.rst [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+.. _pkg.foo:
+
+giflib
+===
+
+-  `Official <http://giflib.sourceforge.net/`__
```
Comment (ruslo) [inline docs/packages/pkg/giflib.rst]: Closing `>` is missing.

File: docs/packages/pkg/giflib.rst [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+-  Added by `sgara <https://github.com/sgara>`__ (`pr-1028 <https://github.com/ruslo/hunter/pull/1028>`__)
```
Comment (ruslo) [inline docs/packages/pkg/giflib.rst]: Pull request `1031`

**Comment by sgara** [comment]
👍 thanks for the comments

**Comment by sgara** [comment]
Done

File: docs/packages/pkg/giflib.rst [hunk 8]
```diff
@@ -0,0 +1,19 @@
+.. spelling::
+
+    foo
```
Comment (ruslo) [inline docs/packages/pkg/giflib.rst]: -> `giflib`

File: docs/packages/pkg/giflib.rst [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+.. index:: media ; giflib
+
+.. _pkg.giflib:
+
+giflib
+===
```
Comment (ruslo) [inline docs/packages/pkg/giflib.rst]: It should be `======` instead of `===`

**Comment by sgara** [comment]
Should be good this time 🤞

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2369
* https://travis-ci.org/ingenue/hunter/builds/275163445?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Some toolchains failed. To exclude broken toolchains please send pull request to branch [pkg.giflib](https://github.com/ingenue/hunter/tree/pkg.giflib), see example:
* https://github.com/ingenue/hunter/pull/56/files

**Comment by sgara** [comment]
I think I'll take a deeper look. Looks like android builds are failing, I'd rather make it work than ignore the tests

**Comment by xsacha** [comment]
See #1152 with CMake version

**Comment by ruslo** [comment]
https://docs.hunter.sh/en/latest/packages/pkg/giflib.html

**Pull request closed without merging**